ga4gh/datamodel/reads.py

```diff
@@ -158,6 +160,9 @@ def getReadAlignments(self, referenceName=None, referenceId=None,
             raise exceptions.BadReadsSearchRequestBothRefs()
         if referenceId is not None:
             referenceName = self._samFile.getrname(referenceId)
+        referenceName, start, end = \
```

Comment:
I kinda liked @macieksmuga's idea of a mixin here. If we use the mixin idea we could have
```python
referenceName, start, end = self.sanitizeAlignmentFileFetch(
     referenceName, start, end)
```
which is a lot neater I think. By defining it as a mixin we can use the class where ever we want.